components/status_dropdown/__snapshots__/status_dropdown.test.tsx.snap

```diff
@@ -471,14 +471,14 @@ exports[`components/StatusDropdown should match snapshot with custom status and
                   className="dndTime-tomorrow_timestamp"
                 >
                   <FormattedDate
-                    value={2021-11-03T22:48:57.000Z}
+                    value={1970-01-02T08:00:02.021Z}
```

Comment:
This should not change to `1970-01-02T08:00:02.021Z`, I think it have something to do with this fake date setter 
https://github.com/mattermost/mattermost-webapp/blob/780bbe87b2b40de0cbb6359648a90aae672f5ff5/tests/helpers/date.ts#L27
@calebroseland any idea?

Reply:
`startOf('day')` seems to be causing the issue (removing it keeps the snapshot dates in `2021`). I'm unsure of how `fakeDate`'s modification of `global.Date` affects `moment`, or why the `startOf` method causes an issue and `add` doesn't—but in any case, we're trying to move away from `moment`.

1. Recommended: instead of `.startOf('day').add('hours', 32)`, you could try `.add(1, 'day').set('hour', 8)`—moment's `add` doesn't seem to interfere with `fakeDate`.
2. Long term: refactor to use `luxon` (probably too large of an effort for this PR, but perhaps `luxon` has better interop with `fakeDate` or another like-solution). `moment` -> `luxon` is the long-term goal.
3. You could find a replacement solution for `fakeDate`.